packages/material-ui/test/integration/MenuList.test.js

```diff
@@ -1,10 +1,23 @@
-import React from 'react';
+import React, { useRef, useLayoutEffect } from 'react';
```

Comment:
For consistency with the codebase:

```suggestion
import React from 'react';
```